lib/base-compiler.ts

```diff
@@ -942,7 +942,7 @@ export class BaseCompiler implements ICompiler {
 
     getDefaultOrOverridenToolchainPath(overrides: ConfiguredOverrides): string {
         for (const override of overrides) {
-            if (override.value) {
+            if (override.name !== CompilerOverrideType.env) {
```

Comment:
There could be a behavioral change if the goal of `if(override.value)` is to also check for empty strings. Is this a concern?

Reply:
sort of, I believe it shouldn't be possible from the UI, but if it's empty the default should be used and not the empty value